protos/foundation/protos/doc/entity.proto

```diff
@@ -166,6 +185,7 @@ message Address {
 message Cluster {
```

Comment:
I feel like NER tools are going to want to work with a pretty raw form of the data, so I don't know how useful our pre-clustering/tokenization will be to them.